src/api.py

```
@@ -89,7 +88,8 @@ def post_drink(jwt):
     try:
         drink = Drink(title=drink_title, recipe=parsed_drink_recipe)
         drink.insert()
-    except BaseException:
+    except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)